apps/web-giddh/src/app/settings/integration/setting.integration.component.ts

```diff
@@ -247,7 +247,9 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
                 if (this.registeredAccount) {
                     this.registeredAccount.map(item => {
                         item.userAmountRanges.map(element => {
-                            element.maxBankLimit = (typeof element.maxBankLimit === "boolean" && element.maxBankLimit) ? 'max' : 'custom';
+                            if (typeof element.maxBankLimit === "boolean") {
```

Comment:
What if (typeof element.maxBankLimit === "boolean") is false?

Reply:
if it get false then it will be string so no need to change it